_posts/2019-10-08-unit-testing-basics.md

```diff
@@ -1138,9 +1138,39 @@ tests, so you can be sure that the piece of code was tested in some capacity.
 Note that code coverage doesn't measure the different behaviors a code path
 **could take**, but only that a particular code path **was taken**.
 
-![]({% img_url unit-testing-basics/code_coverage.png %})
+```
+$ sudo apt install lcov
+$ make lcov
```

Comment:
Wanted to promote that this already setup in the makefile

Reply:
<3